.github/workflows/release.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           replace-with: ""
       - name: Generate JS gRPC bindings
         run: |
-          ./scripts/gen-js-protos.sh ${{steps.makeversion.outputs.replaced}} . ./js-grpc
+          ./scripts/gen-js-protos.sh ${{steps.makeversion.outputs.replaced}} ./proto ./js-grpc
```

Comment:
`./proto/` is now the "root" of our proto files... `./proto/` itself is not part of the proto path, it's just the container for all protos and the actual proto paths are relative to this root.